src/unread/__tests__/unreadSelectors-test.js

```diff
@@ -34,22 +34,22 @@ const unreadStreamData = [
 const unreadPmsData = [
   {
     sender_id: 0,
-    unread_message_ids: [1, 2],
+    unread_message_ids: [11, 12],
   },
   {
     sender_id: 2,
-    unread_message_ids: [3, 4, 5],
+    unread_message_ids: [13, 14, 15],
   },
 ];
 
 const unreadHuddlesData = [
   {
     user_ids_string: '1,2,3',
-    unread_message_ids: [1, 2],
+    unread_message_ids: [21, 22],
   },
   {
-    user_ids_string: '4,5',
-    unread_message_ids: [3, 4, 5],
+    user_ids_string: '1,4,5',
```

Comment:
`unread tests: Fix some impossible data.`

Hmm, I don't yet follow why `'4,5'` (changed to `'1,4,5'`) is impossible for `user_ids_string` here.

Reply:
Ah -- it's because in this data structure, the key for a group PM thread includes all users including self. So in particular there's some user ID (namely self's) that is in every key in the map. So there can't be both `1,2,3` and `4,5` as keys.

I'll add a remark about this.

Reply:
Ah OK, got it, thanks! I thought it might have something to do with the self-user, but I couldn't find anything labelling `1` as the self-user's user ID.

It makes sense though now you put it that way; it's impossible to say that `1,2,3` and `4,5` are both valid keys under the constraint that each key must include the self-user, but it _is_ possible for `1,2,3 and 1,4,5` to both be valid keys given that it's possible for `1` to be the self-user's ID.